src/billing/List/CreateBillingProjectControl.tsx

```diff
@@ -0,0 +1,35 @@
+import { ButtonOutline, Icon } from '@terra-ui-packages/components';
+import React from 'react';
+import { MenuButton } from 'src/components/MenuButton';
+import { MenuTrigger } from 'src/components/PopupTrigger';
+import { CloudProvider } from 'src/workspaces/utils';
+
+interface CreateBillingProjectControlProps {
+  showCreateProjectModal: (type: CloudProvider) => void;
+}
+
+export const CreateBillingProjectControl = (props: CreateBillingProjectControlProps) => {
+  return (
+    <MenuTrigger
+      side='bottom'
+      closeOnClick
+      content={
+        <>
+          <MenuButton aria-haspopup='dialog' onClick={() => props.showCreateProjectModal('AZURE')}>
+            Azure Billing Project
+          </MenuButton>
+          <MenuButton aria-haspopup='dialog' onClick={() => props.showCreateProjectModal('GCP')}>
+            GCP Billing Project
+          </MenuButton>
+        </>
+      }
+    >
+      <ButtonOutline aria-label='Create new billing project'>
```

Comment:
Removed the complexity of `createButton` because it was only called with `undefined` as the argument.

https://github.com/DataBiosphere/terra-ui/pull/5028/files?diff=unified&w=1#diff-3be014ea8adc7a416ef5de1b1f4947ed7704ecdf81bf03db59650a22346861c8L14

The ability to pass a `CloudProvider` dated from when we used to not show the dropdown for GCP users and instead the `Create` button directly opened the GCP create billing project modal.